geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml

```
@@ -948,10 +948,11 @@ vars:
   lingua_extractor: {}
 
   content_security_policy_main_default_src_extra: ''
+  # Google Maps uses map.google.com and map.googleapis.com
+  # Google Analytics 4 uses googletagmanager.com
```

I think that the links are also a good ressource =>
```suggestion
  # Google Maps uses map.google.com and map.googleapis.com
  # More info: https://developers.google.com/maps/documentation/javascript/content-security-policy?hl=fr#allowlist_csp
  # Google Analytics 4 uses googletagmanager.com
  # More info: https://developers.google.com/tag-platform/tag-manager/csp?hl=fr#google_analytics_4_google_analytics
```

---

Ok, done.